addons/fieldmanual/stringtable.xml

```diff
@@ -0,0 +1,359 @@
+            <English>Tow With Ease</English>
+        </Key>
+        <Key ID="STR_ACE_FieldManual_Items_Ropes_Description">
+            <English>%3Ropes%4 have multiple uses including %3Towing%4 vehicles and %3Fast Roping%4 from helicopters.&lt;br/&gt;&lt;br/&gt;%3Towing:%4&lt;br/&gt;%2Approach a vehicle.&lt;br/&gt;%2Use [%3%12%4] and select %3Towing%4.&lt;br/&gt;%2Select rope length.&lt;br/&gt;%2Select attachment point on towing vehicle.&lt;br/&gt;%2Select attachment on towed vehicle.&lt;br/&gt;&lt;br/&gt;%3Available Rope Lengths:%4&lt;br/&gt;%2 3.2 meters&lt;br/&gt;%2 6.2 meters&lt;br/&gt;%2 12.2 meters&lt;br/&gt;%2 15.2 meters&lt;br/&gt;%2 18.3 meters&lt;br/&gt;%2 27.4 meters&lt;br/&gt;%2 36.6 meters</English>
+        </Key>
+        <Key ID="STR_ACE_FieldManual_Items_Sandbag_DisplayName">
+            <English>Sandbags</English>
+        </Key>
+        <Key ID="STR_ACE_FieldManual_Items_Sandbag_ShortName">
+            <English>Expand Your Fortifications</English>
+        </Key>
+        <Key ID="STR_ACE_FieldManual_Items_Sandbag_Description">
+            <English>%3Sandbags%4 are sacks made of sturdy material, filled with sand, used for a variety of purposes such as creating barriers or providing stability in construction projects. Useful in expanding larger placed fortifications.&lt;br/&gt;&lt;br/&gt;%3Usage:%4&lt;br/&gt;%2Equip a %3Sandbag (Empty)%4.&lt;br/&gt;%2Use [%3%11%4] and select %3Deploy Sandbag%4.&lt;br/&gt;%2Follow on-screen instructions for placement.</English>
+        </Key>
+        <Key ID="STR_ACE_FieldManual_Items_SpareBarrels_DisplayName">
+            <English>Spare Barrels</English>
+        </Key>
+        <Key ID="STR_ACE_FieldManual_Items_SpareBarrels_ShortName">
+            <English>Lower Firearm Temperature</English>
+        </Key>
+        <Key ID="STR_ACE_FieldManual_Items_SpareBarrels_Description">
+            <English>%3Spare Barrels%4 allow a soldier to reduce their weapon's heat significantly. After a short delay, the barrel will be swapped and heat reduced. A soldier may also check the temp of any barrels within their inventory.&lt;br/&gt;&lt;br/&gt;%3Usage:%4&lt;br/&gt;%2Use [%3%11%4] and select %3Equipment%4.&lt;br/&gt;%2Select %3Swap Barrel%4.&lt;br/&gt;%2Resume operation after barrel swap is complete.</English>
+        </Key>
+        <Key ID="STR_ACE_FieldManual_Items_SprayPaint_DisplayName">
+            <English>Spray Paint</English>
+        </Key>
+        <Key ID="STR_ACE_FieldManual_Items_SprayPaint_ShortName">
+            <English>Tag Your Territory</English>
+        </Key>
+        <Key ID="STR_ACE_FieldManual_Items_SprayPaint_Description">
+            <English>%3Spray Paint%4 is used to tag structures with various symbols.&lt;br/&gt;&lt;br/&gt;%3Usage:%4&lt;br/&gt;%2Move close to a wall.&lt;br/&gt;%2Use [%3%11%4] and select %3Tag%4.&lt;br/&gt;%2Choose a symbol.&lt;br/&gt;&lt;br/&gt;%3Available Colors:%4&lt;br/&gt;%2Black&lt;br/&gt;%2Blue&lt;br/&gt;%2Green&lt;br/&gt;%2Red</English>
```

Comment:
Spray Paint can be used on more than just walls.

Reply:
What all can it be used on?

Reply:
Vehicles, the ground, walls, doors. "Surface" should be good enough to cover everything generically.